include/GLFW/glfw3.h

```diff
@@ -2276,6 +2324,10 @@ GLFWAPI void glfwGetWindowFrameSize(GLFWwindow* window, int* left, int* top, int
  *  @errors Possible errors include @ref GLFW_NOT_INITIALIZED and @ref
  *  GLFW_PLATFORM_ERROR.
  *
+ *  @remark @wayland There is no concept of iconification in wl_shell, this
+ *  function will always emit @ref GLFW_PLATFORM_ERROR.  TODO: move to
```

Comment:
I'd prefer the todo be an issue tracker entry and that user-facing documentation describe only what is.